server.js

```diff
@@ -333,3 +334,15 @@ app.get('/features/download/:type', (req, res) => {
         break;
     }
 });
+
+const ALL_CH_STR = ALL_CH.join(',');
+app.get('/features/client-hints/', (req, res) => {
+    res.set('accept-ch', ALL_CH_STR);
```

Comment:
this could have been a meta tag, but I wanted to have one source of truth for CH list (all-ch.js). This also has a cool side effect that if you go to http://localhost:3000/features/client-hints/main.html directly header will NOT be sent showing you what's sent by default

Reply:
> directly header will NOT be sent showing you what's sent by default

Only if you haven't already visited a site that has client hints sent? At least that's what it looked like when I played around with it.

It might be nice to have a page where we never request full CH so we can easily test the default. Maybe make use of one of the extra domains for that (e.g., `www.first-party.site`)?

Reply:
Good call, I'll add an expeception for one domain.